source/restricted_schwarz.cpp

```diff
@@ -95,51 +95,60 @@ void SolverRAS<ValueType, IndexType>::setup_local_matrices(
 
     // default local p size set for 1 subdomain.
     first_row[0] = 0;
-    for (auto p = 0; p < num_subdomains; ++p) {
+    for (auto p = 0; p < num_subdomains; ++p)
+    {
```

Comment:
Just a general question: Are you using `clang-format` ? If so, which version are you using ? I prefer the old bracketing style as that reduces the clutter.

Reply:
Well, I prefer the style where the opening brace is in a new line (makes code easier to read for me). But we can agree on one style.